orchestrator/taste-orchestrator.py

```diff
@@ -820,6 +823,54 @@ def BuildGUIs(guiSubsystems, cflagsSoFar, asn1Grammar):
         os.chdir("../..")
 
 
+def BuildRosBridges(rosBridgeSubsystems, cflagsSoFar, asn1Grammar, acnFile):
+    '''Builds automatically generated ROS bridges'''
+    if rosBridgeSubsystems:
+        g_stageLog.info("Building automatically created ROS bridges")
+    for baseDir in rosBridgeSubsystems:
+        olddir = os.getcwd()
+        if not os.path.isdir(baseDir):
+            panic("No directory %s! (pwd=%s)" % (baseDir, os.getcwd()))
+        # This is for GUI code
+        if not os.path.exists(baseDir + os.sep + baseDir + "_ros_bridge_code.cpp"):
+            panic("ROS bridge generated code did not contain a %s ..." % (baseDir + os.sep + baseDir + "_ros_bridge_code.cpp"))
+        os.chdir(baseDir)
+        mkdirIfMissing("ext")
+        mysystem('for i in * ; do if [ -f "$i" -a ! -e ext/"$i" ] ; then ln -s ../"$i" ext/ ; fi ; done')
+        os.chdir("ext")
+        partitionNameWithoutSuffix = g_fromFunctionToPartition.get(baseDir, None)
+        if partitionNameWithoutSuffix:
+            for k, v in g_distributionNodesPlatform.iteritems():
+                if k.startswith(partitionNameWithoutSuffix):
+                    platform = v[0]
+                    break
+            else:
+                platform = None
```

Comment:
`platform` seems to be set, but not used.

Reply:
Removed for now. Have to check somehow that it runs only on linux. The platform string was set to `PLATFORM_NATIVE` so I couldn't differentiate the OS from that.